configs/dpt/dpt_vit-b16_512x512_160k_ade20k.py

```diff
@@ -0,0 +1,58 @@
+_base_ = [
+    '../_base_/models/dpt_vit-l16.py', '../_base_/datasets/ade20k.py',
```

Comment:
Inherited from `dpt_vit-l16_512x512_160k_ade20k.py` instead of `../_base_/models/dpt_vit-l16.py`.

Reply:
The better inherit order is:
`configs/_base_/models/dpt_vit-l16.py` -->` configs/dpt/dpt_vit-b16_512x512_160k_ade20k.py` --> `configs/dpt/dpt_vit-l16_512x512_160k_ade20k.py`

Reply:
From small model to large model.